interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/Typer.scala

```diff
@@ -198,6 +197,24 @@ private[spel] class Typer(
         case d: TypedDict                    => dictTyper.typeDictValue(d, e).map(toNodeResult)
         case TypedUnion(possibleTypes)       => typeUnion(e, possibleTypes)
         case TypedTaggedValue(underlying, _) => typeIndexer(e, underlying)
+        case TypedObjectTypingResult(fields, _, _) =>
+          def stringValue(s: StringLiteral): String = {
+            s.toString.substring(1, s.toString.length - 1)
+          }
+          val indexKeyOpt = e.childrenHead match {
+            case impl: StringLiteral => Some(stringValue(impl))
+            case _                   => None
+          }
+          val fieldOpt = for {
```

Comment:
Let's call it `fieldIndexedByLiteralStringOpt`

Reply:
Done